docs/ReactQt/CreateNewApp.md

```diff
@@ -2,11 +2,17 @@
 
 To create react-native apps with `Desktop` platform support extended version of `react-native-cli` package should be [installed globally](https://github.com/status-im/react-native-desktop/blob/master/docs/ReactQt/InstallUpdatedReactNativeCLI.md).  
 
-3 easy to follow terminal commands required to create and run new react-native app on your desktop:
+In order to setup the RN-desktop project, execute these terminal commands:
 
 ```
 ~$ react-native init DesktopSampleApp && cd DesktopSampleApp
 ~/DesktopSampleApp$ react-native desktop
+
+```
+Afterwards run these commands in separate shells:
+```
+~/DesktopSampleApp$ npm start
+~/DesktopSampleApp$ node node_modules/react-native/ubuntu-server.js
```

Comment:
@siphiuel  `react-native run-desktop` command is supposed to run `npm start` and `node ubuntu-server.js` processes in separate terminals windows in background (if you already have these processes running you might will see only blinks of new terminals windows terminated on execution errors (since ports used by packager and ubuntu-server already occupied)). https://github.com/status-im/react-native-desktop/blob/master/local-cli/desktop/runDesktop.js#L93
This functionality is not working properly on MacOS most of all.